# Problem Set 5: Probability

Stat140: Introduction to the Ideas and Applications of Statistics 


**Problem set policies:** Please provide concise, clear answers for each question. Note that only writing the result of a calculation (e.g., "SD = 3.3") without explanation is not sufficient. For problems involving `R`, include the code in your solution, along with any plots.
		
*You are encouraged to discuss problems with other students (and, of course, with the course head and the TAs), but you must write your final answer in your own words. Solutions prepared "in committee" are not acceptable. If you do collaborate with classmates on a problem, please list your collaborators on your solution.*

#### Problem 1:

Suppose you are helping the police develop a new test for blood alcohol levels. The potential advantage of the test is that it can be used during a routine traffic stop, but neither you nor the police are sure how accurate the test is. By design, the test should be positive when blood alcohol level is above $0.05\%.$

The test has been used on a large number of subjects where a more expensive, less convenient test is known to give perfectly accurate results. The table below shows the joint distribution for the outcome of the new test and true blood alcohol status:

 |         |  **Alcohol level** | **> 0.05%**  |
 
|------------ | -- | --|

|**Test Result**       |   **Yes**    |         **No** |

|Positive  |        0.08       | 0.30 |

|Negative     |        0.02  | 0.60 |  

In [3]:
alcohol.test<-data.frame("Result" =  c("Positive","Negative"), "Greater Than 0.05"=c(0.08,0.02), "No Greater Than 0.05"=c(0.3,0.6))
head(alcohol.test)

Result   Greater.Than.0.05 No.Greater.Than.0.05
1 Positive 0.08              0.3                 
2 Negative 0.02              0.6

a) What is the probability that a randomly selected driver has both a positive test and a blood alcohol level higher than $0.05\%$?

Answer:
$P(\text{Test+ and Alcohol }> 0.05\%)=Pr(A {\cap} B)=0.08$

b) For a randomly selected driver, what is the probability that the test will be positive?

Answer: 
$P(\text{Positive}) = 0.08+0.3 = 0.38$

c) Among drivers who have a positive test, what is the probability that a driver has a blood alcohol level higher than $0.05\%$?

In [3]:
#Use this for calculations

0.08/0.38

[1] 0.2105263

Answer:
$P(>0.05~|~\text{Positive}) = 0.08/0.38 \approx 21.05\%$

d) Suppose $A$ is the event that a driver has a positive test result and $B$ is the event that the driver has a blood alcohol level above $0.05\%$. Are $A$ and $B$ independent? Justify your answer.

In [4]:
#Use this for calculations



Answer: Because $P(>0.05~|~\text{Positive}) =21.05\% \neq 8\% = P(>0.05)$ These events are not independent.

#### Problem 2:

Suppose that a mandatory drug test has a false negative rate of 0.10; i.e., if an employee is a drug user, the test will incorrectly test negative with probability 0.10 and correctly indicate positive with probability 0.90. Now suppose the company HR department will test 4 employees who are *correctly* suspected to be drug users.

Assume independence between employees' results and find the probability that at least one of the employees will correctly test positive.
	

In [ ]:
#Use this space for calculations



Answer: 

#### Problem 3:

Consider the standard normal distribution with mean $\mu=0$ and standard deviation $\sigma= 1$. 

a) What is the probability that an outcome $z$ is greater than 2.60?

In [5]:
#Answer:
1-pnorm(2.6)

[1] 0.004661188

b) What is the probability that $z$ is less than 1.35?

In [6]:
#Answer:
pnorm(1.35)

[1] 0.911492

c) What is the probability that $z$ is between -1.70 and 3.10?

In [7]:
#Answer:
pnorm(3.1)-pnorm(-1.7)

[1] 0.9544669

d) What value of $z$ cuts off the upper 15% of the standard normal distribution?

In [8]:
#Answer:
-qnorm(0.15)

[1] 1.036433

e) What value of $z$ marks off the lower 20% of the distribution?

In [9]:
#Answer:
qnorm(0.2)

[1] -0.8416212

#### Problem 4:

The World Health Organization defines osteoporosis in young adults as a measured bone mineral density 2.5 or more standard deviations below the mean for young adults. Assume that bone mineral density follows a normal distribution in young adults. What percentage of young adults suffer from osteoporosis according to this criterion? Put your answer in terms of the question.


In [11]:
#Use this area for calculations

pnorm(-2.5)

[1] 0.006209665

Answer: About $0.62\%$ of young adults suffer from osteoporosis.